src/core/sys/posix/dirent.d

```diff
@@ -206,7 +236,15 @@ else
 }
 
 int     closedir(DIR*);
-DIR*    opendir(in char*);
+version( NetBSD )
+{
+    DIR*    __opendir30(in char*);
+    alias __opendir30 opendir;
+}
+else
+{
+    DIR*    opendir(in char*);
+}
```

Comment:
The usual penance if you discover a discrepancy on one platform is to write a version block for each known tested platform, using ``static assert (0)`` for the else branch.

This alerts the next person who will port this code to go check and verify that it's defined properly.